package Exception3;

public class Exception2 {
    public static void main(String[] args) {

        // Runtime exception - error in program
        int a = 1 / 0;

        String name = null;
        name.length();

        int[] arr = new int[2];
        System.out.println(arr[2]);
    }
}
